tests/pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+markers =
+    submodule_multiple_techniques: test parsing of EC-lab generated files (deselect with '-m "submodule_multiple_techniques"')
```

Comment:
I think we need to flip these around. A) I don't think the mark needs to be that specific, I would suggest just "submodule" or "extended_tests" and I would suggest that they are deselected unless specifcally enabled. The purpose would be, that any developer should be able to make a PR running the standard tests without all the submodules and then the full suite will be run only before release or selectively by developers needing them.